startup/30-basler.py

```diff
@@ -0,0 +1,118 @@
+import datetime
+import itertools
+
+import matplotlib.pyplot as plt
+import numpy as np
+from pypylon import pylon
+
+from ophyd import Component as Cpt
+from ophyd import Device, Signal
+from ophyd.sim import NullStatus
+
+import os
+
+os.environ['PYLON_CAMEMU'] = "1"
+
+plt.ion()
+
+class BaslerCamera(Device):
+
+    image = Cpt(Signal, kind="normal")
+    mean  = Cpt(Signal, kind="hinted")
+    shape = Cpt(Signal, kind="normal")
+
+    def __init__(self, name='basler_cam',
+                    sim_id=None, watch_name=None,
+                    sirepo_server='http://10.10.10.10:8000', source_simulation=False,
+                    root_dir='/tmp/sirepo_det_data', **kwargs):
```

Comment:
The args/kwargs need to be revised as they are not relevant to this implementation.